spec/forms/employment_form_spec.rb

```diff
@@ -98,13 +98,13 @@
     context 'when the employment model has not been persisted' do
       before { allow(employment).to receive_messages persisted?: false }
 
-      it 'is false' do
-        expect(employment_form.was_employed).to be false
+      it 'is nil' do
+        expect(employment_form.was_employed).to be_nil
       end
     end
 
     context 'when the employment model has been persisted' do
-      before { allow(employment).to receive_messages persisted?: true }
```

Comment:
Maybe change the name of this context as it no longer has anything to do with if the employment model is persisted ?